internal/MakeR8Set.ecl

```diff
@@ -26,15 +26,12 @@ EXPORT SET OF REAL8 makeR8Set(dimension_t r, dimension_t s,
                               DATASET(Layout_Cell) D,
                               BOOLEAN transpose=FALSE) := BEGINC++
     // copy of Layout_Cell translated to C
-    #ifndef pbb_cell
-        typedef struct __attribute__ ((__packed__)) {
-          uint16_t wi_id;
-          uint32_t x;
-          uint32_t y;
-          double v;
-          } pbb_cell;
-    #endif // pbb_cell    #ifndef pbb_cell
-
+    struct __attribute__ ((__packed__)) pbb_cell{
```

Comment:
I am not an expert at C++ but I am under the impression that the struct declaration should be wrapped with an if not defined (#ifndef FOO and #define FOO).  If this structure is used (and it could be in my opinion) by another BEGINC++ attribute, then you could get a multiple defined name.  @richardkchapman  should correct me if multiple definitions of the same struct are OK.

Reply:
I woul dhave thought simply removing the #body on line 35 (meaning that the struct definition would be local to the generated function) would be the simplest and cleanest way to avoid that potential issue.

Reply:
Sounds OK to me, but I don't have exact knowledge of the visibility rules for the modern version of the C++ language.